src/mfoc.c

```diff
@@ -910,7 +910,7 @@ get_rats_is_2k(mftag t, mfreader r)
     exit(EXIT_FAILURE);
   }
   if (res >= 10) {
-    printf("ATS %02X%02X%02X%02X%02X|%02X%02X%02X%02X\n", res, abtRx[0], abtRx[1], abtRx[2], abtRx[3], abtRx[4], abtRx[5], abtRx[6], abtRx[7], abtRx[8]);
+    printf("ATS %02X%02X%02X%02X%02X|%02X%02X%02X%02X%02X\n", res, abtRx[0], abtRx[1], abtRx[2], abtRx[3], abtRx[4], abtRx[5], abtRx[6], abtRx[7], abtRx[8]);
```

Comment:
This change is unrelated

Reply:
ok,  changed title.

Reply:
D'oh, I though it was a mistake… What warning is this change fixing?

Reply:
Never mind… I have to read what you have already written…

Reply:
now,  I was thinking that you would comment that the added formatter was in the wrong place...
Since I basically just added it to the end,

Reply:
I have no idea what this code is supposed to do :smiley:  Nevertheless, 10 `%x` in a `printf()` require 10 arguments.